src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/how-to-upgrade-from-lts-2023-to-lts-2025/how-to-upgrade-audit-service.md

```diff
@@ -0,0 +1,255 @@
+    <artifactId>nuxeo-platform-audit-test</artifactId>
+    <scope>test</scope>
+  </dependency>
+  ..
+</dependencies>
+```
+
+## Java Code
+
+All services or POJO classes you may use from the `org.nuxeo.ecm.platform.audit` package have migrated to `org.nuxeo.audit` 
+or have been replaced. 
+This mainly impacts the following usages:
+
+- `org.nuxeo.ecm.platform.audit.api.LogEntry` should be updated to `org.nuxeo.audit.api.LogEntry`
+- usage of `AuditAdmin`, `AuditLogger`, `AuditReader` and `Logs` should be updated to `org.nuxeo.audit.service.AuditBackend`
+
+For example, previous Audit Service usage would be:
+
+```java
+import org.nuxeo.ecm.platform.audit.api.AuditLogger;
+import org.nuxeo.ecm.platform.audit.api.LogEntry;
+
+protected void addAudit(AuditLogger logger, String auditComment, String principalName) {
+    LogEntry logEntry = logger.newLogEntry();
+    logEntry.setEventId("an event id");
+    logEntry.setCategory("an event category");
+    logEntry.setEventDate(new Date());
+    logEntry.setPrincipalName(principalName);
+    logEntry.setComment(auditComment);
+    logger.addLogEntries(Collections.singletonList(logEntry));
+}
+```
+
+This might be converted to:
+
+```java
+import org.nuxeo.audit.api.LogEntry;
+import org.nuxeo.audit.service.AuditBackend;
+
+protected void addAudit(String auditComment, String principalName) {
+    var logEntry = LogEntry.builder("an event id", new Date())
+                            .category("an event category")
+                            .principalName(principalName)
+                            .comment(auditComment)
+                            .build();
+    Framework.getService(AuditBackend.class).addLogEntries(List.of(logEntry));
+}
+```
+
+## XML Contributions
+
+Contributions made to `org.nuxeo.ecm.platform.audit.service.NXAuditEventsService` extension points should be updated to the new services.
+
+### Events
+
+Events extension point declaration didn’t change, just the target of the contribution:
+
+```xml
+<extension target="org.nuxeo.ecm.platform.audit.service.NXAuditEventsService" point="event">
+    <event name="eventNameToAudit" />
+</extension>
+```
+
+Should be updated to:
+
+```xml
+<extension target="org.nuxeo.audit.service.AuditComponent" point="event">
+    <event name="eventNameToAudit" />
+</extension>
+```
+
+### Extended Info
+
+Extended Info extension point didn’t change, just the target of the contributions:
+
+```xml
+<extension target="org.nuxeo.ecm.platform.audit.service.NXAuditEventsService" point="extendedInfo">
+    <extendedInfo key="title" expression="${source.dublincore.title}" />
+</extension>
+```
+
+Should be updated to:
+
+```xml
+<extension target="org.nuxeo.audit.service.AuditComponent" point="extendedInfo">
+    <extendedInfo key="title" expression="${source.dublincore.title}" />
+</extension>
+```
+
+## Operation
+
+The `Audit.Query` operation has been removed from contributed operations because it was tied to SQL Audit Backend.
```

Comment:
That means it still there but not contributed? Maybe add a comment on how to re-enable it if **really needed**?